### ISSUE TITLE
GIFv breaks result storage

### ISSUE THREAD

**Issue by cartrdgeproduct** [thumbor/thumbor#848]
### Thumbor request URL
`http://localhost:8000/unsafe/filters:gifv(mp4)/https://somedomain.com/big.gif`

### Expected behaviour
On the second or any subsequent request, the result should be fetched from result storage without error.

### Actual behaviour
First request succeeds. All later requests fail when results storage is enabled.

### Operating system
Debian

### Your thumbor.conf
```
DETECTORS=['thumbor.detectors.face_detector','thumbor.detectors.profile_detector','thumbor.detectors.feature_detector']
FILTERS=['thumbor.filters.format','thumbor.filters.extract_focal','thumbor.filters.no_upscale']
HTTP_LOADER_REQUEST_TIMEOUT=120
MAX_AGE=31557600
OPTIMIZERS=['thumbor.optimizers.gifv']
STORAGE=thumbor.storages.no_storage
THUMBOR_WORKER_COUNT=1
USE_GIFSICLE_ENGINE=True
WEBP_QUALITY=90
LOG_PARAMETER=-l info
AWS_ACCESS_KEY_ID=AN_ID
AWS_SECRET_ACCESS_KEY=A_KEY
TC_AWS_REGION=A_REGION
TC_AWS_ENDPOINT=AN_ENDPOINT
TC_AWS_RESULT_STORAGE_BUCKET=A_BUCKET
TC_AWS_RESULT_STORAGE_ROOT_PATH=A_DIRECTORY
RESULT_STORAGE=tc_aws.result_storages.s3_storage
RESULT_STORAGE_STORES_UNSAFE=True
```

Thumbor fails to load GIFv results from S3. The first request works perfectly and regular images like jpegs work as expected.

Thumbor version 6.2.0 via APSL docker use the `tc_aws.result_storages.s3_storage` result storage.

An example request:
`curl -I "http://localhost:8000/unsafe/filters:gifv(mp4)/https://somedomain.com/big.gif"`

First response:
```
2017-01-05 06:19:15 tornado.access:INFO 200 HEAD /unsafe/filters:gifv(mp4)//https://somedomain.com/big.gif (172.17.0.1) 10220.31ms
```

All subsequent responses:
```
2017-01-05 06:28:47 tornado.application:ERROR Future exception was never retrieved: Traceback (most recent call last):
  File "/usr/local/lib/python2.7/site-packages/tornado/gen.py", line 1024, in run
    yielded = self.gen.send(value)
  File "/usr/local/lib/python2.7/site-packages/thumbor/handlers/__init__.py", line 117, in execute_image_operations
    self.context.request.engine.load(buffer, EXTENSION.get(mime, '.jpg'))
  File "/usr/local/lib/python2.7/site-packages/thumbor/engines/pil.py", line 312, in load
    super(Engine, self).load(buffer, self.extension)
  File "/usr/local/lib/python2.7/site-packages/thumbor/engines/__init__.py", line 162, in load
    image_or_frames = self.create_image(buffer)
  File "/usr/local/lib/python2.7/site-packages/thumbor/engines/pil.py", line 70, in create_image
    img = Image.open(BytesIO(buffer))
  File "/usr/local/lib/python2.7/site-packages/PIL/Image.py", line 2319, in open
    % (filename if filename else fp))
IOError: cannot identify image file <_io.BytesIO object at 0x7fd77b692710>
```

I'm using the APSL docker image which you can easily run with:
`docker run -p 8000:8000 --env-file .env "apsl/thumbor"`

^ You just need to create a .env file, adding valid credentials for an S3 bucket. Unsafe is optional.

Note that the transformed result (an mp4) is in fact sitting in S3 so it did store correctly. It just fails to recognize the format upon retrieval.

**Comment by kkopachev** [comment]
This might be fixed in 6.2.1 via https://github.com/thumbor/thumbor/pull/839

**Comment by cartrdgeproduct** [comment]
Confirmed, it is fixed in version 6.2.1
